### ISSUE TITLE
I can't visit any website when I turn JustTrustMe on.

### ISSUE THREAD

**Issue by tanghengvip** [Fuzion24/JustTrustMe#47]
Everytime I turn JustTrustMe on. 
Then all webview(any app or browser) can't visit any webpage and show network errors.
It is strange that I can sniffer the App's API requests as normal at the same time. 
I thought it was the problem of my device or it's webview. then I try other devices. all the same.
Does anyone got same situation? how to fix it?

**Comment by lurumdare** [comment]
Install Fiddler certificate?

**Comment by tanghengvip** [comment]
> Install Fiddler certificate?

Yes. I did.  HTTPS and API requests are working fine with Fiddler but webview.

**Comment by lurumdare** [comment]
@tanghengvip And what did you sniff?

**Comment by tanghengvip** [comment]
> @tanghengvip And what did you sniff?

Like a App including Native&Webview. The webview part always fail. and I saw people got same problem  on Baidu(Chinese Google^_^).

**Comment by ASheaJin** [comment]
> > @tanghengvip And what did you sniff?
> 
> Like a App including Native&Webview. The webview part always fail. and I saw people got same problem on Baidu(Chinese Google^_^).

I also encounter such a problem. Can you solve it now?Tks

**Comment by tanghengvip** [comment]
> > > @tanghengvip And what did you sniff?
> > 
> > 
> > Like a App including Native&Webview. The webview part always fail. and I saw people got same problem on Baidu(Chinese Google^_^).
> 
> I also encounter such a problem. Can you solve it now?Tks

Try this! (somebody tell me they fixed it and build this version):
https://mega.nz/#!XrhESCxL!7UQ8DhIX65MxqsCUFw1Q3d0ILu865LyyfZzd_aF0jQU

**Comment by gokaybiz** [comment]
> > > > @tanghengvip And what did you sniff?
> > > 
> > > 
> > > Like a App including Native&Webview. The webview part always fail. and I saw people got same problem on Baidu(Chinese Google^_^).
> > 
> > 
> > I also encounter such a problem. Can you solve it now?Tks
> 
> Try this! (somebody tell me they fixed it and build this version):
> https://mega.nz/#!XrhESCxL!7UQ8DhIX65MxqsCUFw1Q3d0ILu865LyyfZzd_aF0jQU

Is this safe to use?
After run this build, my calender infected with spam messages.

**Comment by tanghengvip** [comment]
> > > > > @tanghengvip And what did you sniff?
> > > > 
> > > > 
> > > > Like a App including Native&Webview. The webview part always fail. and I saw people got same problem on Baidu(Chinese Google^_^).
> > > 
> > > 
> > > I also encounter such a problem. Can you solve it now?Tks
> > 
> > 
> > Try this! (somebody tell me they fixed it and build this version):
> > https://mega.nz/#!XrhESCxL!7UQ8DhIX65MxqsCUFw1Q3d0ILu865LyyfZzd_aF0jQU
> 
> Is this safe to use?
> After run this build, my calender infected with spam messages.

Are you sure that spam is related to this? I have not encountered a similar problem.check it again? btw. did it solve your problem?

**Comment by tanghengvip** [comment]
> > > @tanghengvip And what did you sniff?
> > 
> > 
> > Like a App including Native&Webview. The webview part always fail. and I saw people got same problem on Baidu(Chinese Google^_^).
> 
> I also encounter such a problem. Can you solve it now?Tks

You should try this [solotion](https://blog.jeroenhd.nl/article/android-7-nougat-and-certificate-authorities)

**Comment by tanghengvip** [comment]
> > > @tanghengvip And what did you sniff?
> > 
> > 
> > Like a App including Native&Webview. The webview part always fail. and I saw people got same problem on Baidu(Chinese Google^_^).
> 
> I also encounter such a problem. Can you solve it now?Tks

https://mega.nz/#!uvp1xCJJ!CD_D27Fy9OIW60Bf0GPKN14xf50lKVXG2Tt298RjW8E
Use this one!!! everything will be ok.

**Comment by yoshimo** [comment]
I still get trouble with Google components in a boringssl library. Is that supposed to work with v4 or do we need additional hooks?

**Comment by tanghengvip** [comment]
> I still get trouble with Google components in a boringssl library. Is that supposed to work with v4 or do we need additional hooks?

Does normal Apps work with v4?

**Comment by yoshimo** [comment]
So far yes

**Comment by GGGON** [comment]
> > > > @tanghengvip And what did you sniff?
> > > 
> > > 
> > > Like a App including Native&Webview. The webview part always fail. and I saw people got same problem on Baidu(Chinese Google^_^).
> > 
> > 
> > I also encounter such a problem. Can you solve it now?Tks
> 
> https://mega.nz/#!uvp1xCJJ!CD_D27Fy9OIW60Bf0GPKN14xf50lKVXG2Tt298RjW8E Use this one!!! everything will be ok.

thanks
